src/Numerics/GridBConds.h

```diff
@@ -84,15 +86,8 @@ struct GridBCond
   {
     if(periodic)
     {
-#if defined(HAVE_STD_ROUND)
       T x1=x*OneOverL;
       x=L*(x1-round(x1));
```

Comment:
I thought we had solved the problem in /src/config/stdlib/math.h, I rerouted the round there.